Percy/Percy.cs

```diff
@@ -67,6 +72,10 @@ private static dynamic Request(string endpoint, object? payload = null, bool isJ
         {
             StringContent? body = payload == null ? null : new StringContent(
                 PayloadParser(payload, isJson), Encoding.UTF8, "application/json");
+            if (!IsRunningFromXUnit) {
+                _http = new HttpClient();
+                _http.Timeout = TimeSpan.FromMinutes(10);
+            }
```

Comment:
why?

Reply:
tests are failing because we are assigning new _http instance

Reply:
we should fix testcase in that case.